aip_x2_launch/launch/lidar.launch.xml

```diff
@@ -7,6 +7,7 @@
   <arg name="use_pointcloud_container" default="false" description="launch pointcloud container"/>
   <arg name="pointcloud_container_name" default="pointcloud_container"/>
   <arg name="dual_return_filter_param_file" default="$(find-pkg-share aip_x2_launch)/config/dual_return_filter.param.yaml"/>
+  <arg name="enable_blockage_diag" default="true"/>
```

Comment:
[Q]
This enable_blockage_diag argument doesn't appear to be used in the x2 launch (in panda_node_container). Is it intended not to be used?